pages/03.Howtos/15.repartition-vm-disk/docs.en.md

```diff
@@ -0,0 +1,255 @@
+---
+title: 'How to repartition VM disk'
+date: '21-10-2020 16:27'
+taxonomy:
+    category:
+        - docs
+---
+
+## How to repartition VM disk
+
+### Overview
+
+This Document will show you the essential steps to repartition your VM disk. For this purpose we are using an Ubuntu 18.04 server as example with the `m1c.tiny` [flavor](../../04.Reference/03.compute/docs.en.md).
+
+
+### Prerequisites
+
+* You need to have the login data for the SysEleven Stack API (user name and passphrase)
+* Knowledge how to utilise a terminal/SSH and SSH-keys.
+
+### Automatically configure partitions
+
```

Comment:
Perhaps add here that we recommend the automatic way for production use and/or that the manual method is meant more for instructional use than for production.

I like the script you made!

Reply:
The honor belongs to @frittenlab / MetaKube guys, I gave up struggling with cloudinit in the process of os-7522. This approach seems the best suitable =)